### PR TITLE
[IIIF-828] Improve Festerize description

### LINKED ISSUE #102: Clarify Readme.md
For me the use of the word "default" in the readme (and in the help) implies that if you like the default, then you don't need to specify. Instead, in this instance I think the better word might be "expected" or "most usual".

### PR DESCRIPTION
**Pull request by markmatney**
fixes #102

### CODE REVIEW

File: src/main/scripts/festerize/README.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 Uploads CSV files to the Fester IIIF manifest service.
 
+After Fester processes the CSVs in order to create or update any IIIF collections or manifests, it updates and returns those CSV files to the user.
```
Comment (emcaulay) [inline src/main/scripts/festerize/README.md:5]: If the CSV contains a work that has never been festerized before, then a column is added to the CSV that provides the IIIF Manifest URL. If the CSV contains pages, the values in the CSV are used to expand or revise a previously festerized work record.

Comment (emcaulay) [reply src/main/scripts/festerize/README.md:5]: That's my understanding of what happens. I know you have that in the next paragraph to some extent so you might want to revise to merge my comment with your description.

File: src/main/scripts/festerize/README.md [hunk 2]
```diff
@@ -24,6 +28,12 @@ Usage: festerize [OPTIONS] SRC...
 
   Uploads CSV files to the Fester IIIF manifest service.
```
Comment (emcaulay) [inline src/main/scripts/festerize/README.md:29]: I don't understand this paragraph and how it relates to the beginning of the file. It is possible I'm misreading the Github display. But I think this repeats the same heading as earlier in the readme.

Comment (markmatney) [reply src/main/scripts/festerize/README.md:29]: This is just a copy of what's at the beginning of the file (as well as the description in the Python script itself).

At some point we decided to put the entire "help" output inside the README, but whether or not that is useful is open for discussion.

Comment (ksclarke) [reply src/main/scripts/festerize/README.md:29]: Yes, that's a question we should ask again... @emcaulay, we could just have the read me say, "See help output for use"... Is there value putting documentation on how to use the script in the README or would you prefer just a pointer to the "help" output?

**Pull request closed without merging**